src/host/srvinit.cpp

```diff
@@ -295,6 +311,83 @@ void ConsoleCheckDebug()
     return S_OK;
 }
 
+[[nodiscard]] HRESULT ConsoleEstablishHandoff(_In_ HANDLE Server,
+                                              HANDLE driverInputEvent,
+                                              PCONSOLE_API_MSG connectMessage)
+try
+{
+    auto& g = ServiceLocator::LocateGlobals();
+    g.handoffTarget = true;
+
+    IID delegationClsid;
+    if (SUCCEEDED(DelegationConfig::s_GetDefaultConsoleId(delegationClsid)))
+    {
+        g.handoffConsoleClsid = delegationClsid;
+    }

```

Comment:
if we're about to hand off to another terminal, then do we need to set up `handoffConsoleClsid`?

Reply:
No, not strictly. I could guard this so only the inside windows one looks up the console and the outside one looks up the terminal.

Reply:
meh, that's fine.